### PR TITLE
Extended Developer Program pages

### PR DESCRIPTION
**Pull request by calmrat**
Rebased off the mattg-gaming-pr, this PR includes additional pages that cover the Extended Developer Program.

### CODE REVIEW

**Comment by calmrat** [comment]
There is one additional link missing i'll need to update before applying this rebase; I will share a comment here once's it's in place

**Comment by calmrat** [comment]
@AndreiNavoichyk any chance for a review? my first commits, wanna make sure everything looks alright... i incorporated comments from fred.

File: edp/bounties.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Examples
+
+Some examples of the bounties include:
+
+- Technical
+  - preparing a deployable repository demonstrating a new use-case for (eg) [cere-ddc-sdk-js](https://github.com/Cerebellum-Network/cere-ddc-sdk-js)
+  - running a network simulation for validating that Cere Network slashing works as expected
+  - creating a new community maintained client SDK for a new language (eg, rust)
+  - adding new content to our [user-guide](https://docs.cere.network/ddc/developer-guide/examples)
```
Comment (AndreiNavoichyk) [inline edp/bounties.md:35]: We should use relative links.

File: edp/bounties.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Social
```
Comment (AndreiNavoichyk) [inline edp/bounties.md:36]: Should we have the same bounties types (tech and non-tech or tech and social) in all documentation?

File: edp/contributor-journey.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+- For example, submitting a bug report with steps to reproduce or sharing a unique, personalized review of a project or update in the ecosystem.
+
+### Level 3 - “Contributing”
+
+We expect high-quality work at this level. Something which might be incorporated directly into the project or ecosystem. 
+
+*You should be proficient builder within the Cere Ecosystem and seen as a essential part of the team.* 
+
+- For example, preparing a new example use-case not already covered in the [Developer Guide](https://docs.cere.network/ddc/developer-guide/examples/) or a complete bug/feature request including pull-request.
```
Comment (AndreiNavoichyk) [inline edp/contributor-journey.md:31]: We should use relative links.

File: edp/overview.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+## Extended Developer Program ##
+
+The Extended Developer Program is a Cere backed program offering early joining, tech-savvy individuals, teams and enterprises early access to important changes in the ecosystem. 
+
+Through this program, the Cere Network team engages with developers on-line and IRL at events, collaborating within the community and ecosystem at large to build the future of decentralized data cloud solutions as laid out in the Cere Network [Vision 2.0 whitepaper](https://cere.network/cere-vision-2-0/)
+
+We work with the community to help us test our code, build client SDKs, identify bugs and security vulnerabilities, and build new tools and integrations with other blockchain projects and initiatives. 
+
+We are looking for ecosystem partners to collaborate on funded and incentivized projects with the Cere Community. [Write to us](mailto:chris.ward@cere.network) if it sounds like you!
```
Comment (AndreiNavoichyk) [inline edp/overview.md:19]: What about creating some alias email? For example, `edp@cere.network`.

File: edp/overview.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+## OGs
+Everyone in the community can participate. Elite contributors earn the ability to get early access to support and resources around validator node pool, community admins and MODs, github project comitters, and sometimes Cere Network colleagues! 
+
+We call these elite leads in our community `OG`, like `original-gansgsta`.  They are responsible for helping at all levels of the community. We couldn't work without them, and sometimes even, they get paid for their work. You'll see them around, and know it when you do. 
+
+In the future, OGs will also be a major source of any DAOs Cere EDP community might initiate around the EDP program. Stay tuned!
+
+## Where can you find us? 
+Join the one and only Official EDP Developer Community on 
+- [discord](https://discord.gg/bFv6jkpK8N)
```
Comment (AndreiNavoichyk) [inline edp/overview.md:79]: Can we somehow reuse these links in all docs?

**Comment by AndreiNavoichyk** [comment]
@calmrat 
Please, ask also @svalasovich to review.

**Review by svalasovich — APPROVED** [review]

File: other/gaming.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Cere Gaming
```
Comment (svalasovich) [inline other/gaming.md:1]: We have gaming page in DDC v2 testnet. Don't we need to add it here?

**Review by svalasovich — APPROVED** [review]

File: other/gaming.md (hunk 6 above)
Comment (aelyseev) [reply other/gaming.md:1]: good point

https://games.dev.cere.network/

File: other/gaming.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+Add value instantly to your game by enabling customizable player experiences using Cere’s Freeport and Decentralised Data Cloud. Cere’s Tools and services allow you to:
+
+- Create in-game assets through Cere Freeport creator suite
+- Use the Cere Tools for immersive player experiences
+- No-install browser games can be launched from anywhere
+- But mobile browser-optimized games that bypass app stores and royalty payments
```
Comment (aelyseev) [inline other/gaming.md:8]: sound like a part of single sentence

File: other/gaming.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+- Unlock the power data through Cere’s DDC
+- Wallets that can directly communicate with game packs to handle the end-to-end flows of signup/authentication/payment/custody of assets
+- Track achievements, progress, and digital content acquired in users' wallets. 
+
+**Want to learn how Cere Toolset can supercharge your game?**
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 👉 Reach out to the [Cere Team](https://cere.network/contact-gaming/)!
```
Comment (aelyseev) [inline other/gaming.md:15]: redundant spacing

**Comment by calmrat** [comment]
preparing a new PR that's cleaner

**Pull request closed without merging**